test/data_manager.py

```diff
@@ -0,0 +1,66 @@
+import os
+import shutil
+from collections import defaultdict
+from pathlib import Path
+from typing import List, Dict
+
+
+class DataManager:
```

Comment:
Name too  generic, I've no idea what this is. If you can't come up with something better, add documentation.